src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -246,18 +281,44 @@ class ElectronicDeliveryForm extends React.Component {
               value={this.state.form.endPage}
               onChange={(e) => this.handleUpdate(e, 'endPage')}
             />
-            {
-              errorClass.endPage &&
-                (<span
-                  className="nypl-field-status"
-                  id="end-page-status"
-                  aria-live="assertive"
-                  aria-atomic="true"
-                >
-                  {this.state.error.endPage}
-                </span>)
+            {(errorClass.endPage) &&
+              <span
+                className="nypl-field-status"
+                id="end-page-status"
+                aria-live="assertive"
+                aria-atomic="true"
+              >
+                {this.state.error.endPage}
+              </span>
             }
           </div>
+          <div className={`nypl-text-field ${(errorClass.startPage || errorClass.endPage) ? 'nypl-field-error' : ''}`}>
+            <span
+              className="nypl-field-status"
+              id="page-status"
+              aria-live="assertive"
+              aria-atomic="true"
+            >
+              <span>Values muse be numeric only</span><br />
+              <span>(No special characters allowed)</span>
+            </span>
+          </div>
+        </fieldset>
+        <fieldset className="nypl-fieldset v2 additional-notes">
+          <legend>Additional Notes</legend>
+          <h3>Additional Notes</h3>
+
+          <div className="nypl-text-field">
+            <label htmlFor="request-notes" id="request-notes-label">Additional Notes</label>
+            <input
```

Comment:
This should be a `textarea`. Check https://nypl.github.io/design-toolkit/edd-request-form.html.